src/index.html

```diff
@@ -134,12 +143,63 @@ <h2 class="modal__title">New Post</h2>
         <div class="modal__content">
           <img src="" alt="pic" class="modal__img" />
           <button
-            class="modal__close-btn modal__close-btn_light"
+            class="modal__close-btn modal__close-btn_preview"
             type="button"
           ></button>
           <p class="modal__caption"></p>
         </div>
       </div>
+      <div class="modal" id="delete-modal">
+        <div class="modal__content modal__content_delete">
+          <div class="modal__container modal__container_smaller">
+            <p class="modal__question">
+              Are you sure you want to delete this image?
+            </p>
+            <form class="modal__delete-form">
```

Comment:
Please, add `name="delete-form"` for the delete form in `index.html`, to be able to correctly take it from `document.forms`



```js
            <form class="modal__delete-form" name="delete-form">

```